docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -331,8 +331,7 @@ README](https://github.com/microservices-demo/microservices-demo).
 Note that the Sock Shop demo only works on `amd64`.
 
 ``` bash
-kubectl create namespace sock-shop
-kubectl apply -n sock-shop -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
+kubectl apply -f "https://github.com/microservices-demo/microservices-demo/blob/master/deploy/kubernetes/complete-demo.yaml?raw=true"
```

Comment:
Looking at that yaml, it looks like it doesn't actually create the namespace.  Do we perhaps need the starting `create namespace` command still?